sonar-core/src/main/java/org/sonar/core/config/CorePropertyDefinitions.java

```diff
@@ -218,10 +218,12 @@ private CorePropertyDefinitions() {
       PropertyDefinition.builder(CoreProperties.CPD_CROSS_PROJECT)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "CPD_CROSS_PROJECT"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)